pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -252,6 +255,13 @@ let
       proprietary_codecs = true;
       enable_hangout_services_extension = true;
       ffmpeg_branding = "Chrome";
+    } // optionalAttrs useOzoneWayland {
```

Comment:
Is chromium going to be mutual exclusive with x11 support?

Reply:
No, Ozone does not mean any sort of mutual exclusivity, and the GN flags here explicitly enable both the x11 and wayland backends for Ozone. The toggle here is really `useOzone`: ie, `useOzone=false` means the existing shipping Chromium display tech (GTK and X11), and `useOzone=true` means to switch to the new Chromium display stack (Ozone) and enable both the X11 and Wayland backends for it.